consensus/core/src/tx.rs

```diff
@@ -347,6 +347,11 @@ impl MutableTransaction {
         Self { tx, entries: entries.into_iter().map(Some).collect(), calculated_fee: None, calculated_mass: None }
     }
 
+    pub fn as_verifiable(&self) -> impl VerifiableTransaction + '_ {
+        debug_assert!(self.is_verifiable());
```

Comment:
nit: I think it should be a regular `assert`

Reply:
Done.